CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Use pipenv for python dependency management. #1091
 - Moved unit tests to a dedicated `tests/` directory to improve pytest
   collection time. #1102
+- Added `--run-performance-tests` flag to BB tests. If this flag is not specified,
```

Comment:
Either 

1. drop the word "Added" and move this to the `Added` section above

or

2. Reword as a change (e.g. "Performance tests are skipped if BB tests aren't run with the `--run-performance-tests` flag")

Reply:
I'd like to keep "Added" for new features. Skipping performance tests is nothing new functionally, just a convenience thing.